pom.xml

```diff
@@ -148,7 +148,7 @@
     <httpcomponents.client.version>4.5.13</httpcomponents.client.version>
     <httpcomponents.core.version>4.4.13</httpcomponents.core.version>
     <ivy.version>2.5.1</ivy.version>
-    <jackson.version>2.12.7</jackson.version>
+    <jackson.version>2.12.7.1</jackson.version>
```

Comment:
You are confused with jackson-databind version and jackson bom version, 2.12.7.1 is the version for jackson-databind 
https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind/2.12.7.1

Reply:
Thanks @ayushtkn  for pointing out, will raise another PR with the right version.